src/components/periodic-table-page/PeriodicTablePage.tsx

```diff
@@ -10,102 +8,90 @@ import PeriodicTable, {
   IPeriodicTableElement,
 } from "../periodic-table/PeriodicTable";
 import PtElementInfo from "../pt-element/PtElementInfo";
-import Modal from "../shared/modal/Modal";
 import Navbar from "../shared/navbar/Navbar";
 import SwipeableModal from "../shared/swipeable-modal/SwipeableModal";
 import ElementInfo from "./element-info/ElementInfo";
 import "./PeriodicTablePage.scss";
 
-type Props = RouteComponentProps<any> & React.Props<any>;
-
 interface IPeriodicTablePageState {
```

Comment:
Not necessary, but the repo owner has stated in other PRs that he would prefer the `I...` prefix on interfaces be dropped where we're making changes.